### PR TITLE
docs(website): beta banner

### PR DESCRIPTION
**Pull request by redox**
screen shot 2016-07-30 at 14 06 22

### CODE REVIEW

**Comment by rayrutjes** [comment]
Wow nice! Let me check that out!

**Comment by rayrutjes** [comment]
I see 2 small problems:

1. We should add some more padding to the header:
image

2. We should add some padding to the H2 anchor trick, so that when we follow an anchor the title is not overlapped
image


I can fix this if you'd like, let me know.

**Comment by redox** [comment]
Arg didn't see that; on it.

**Comment by redox** [comment]
@rayrutjes fixed, with the CSS anchoring mess \o/

File: docs/src/sass/app.scss [hunk 1]
```diff
@@ -158,7 +158,7 @@ table thead tr, #content table tbody tr {
 
 /* Header */
 #top-section {
- padding: 1.3rem 0 0.2rem;
+ padding: ($navigation-height + $beta-banner-height) 0 0.2rem;
```
Comment (rayrutjes) [inline docs/src/sass/app.scss]: I would remove the `$beta-banner-height` here, 50px  padding top looks better from my POV.

File: docs/src/sass/app.scss [hunk 2]
```diff
@@ -193,13 +193,24 @@ table thead tr, #content table tbody tr {
 }
 
 #content {
- h2, h3, h4, h5, h6 {
-  margin-top: 5rem;
```
Comment (rayrutjes) [inline docs/src/sass/app.scss]: I'd love to keep some more spacing between headings. WDYT?

**Comment by rayrutjes** [comment]
GG for the anchor mess refactoring 🎢

File: docs/src/sass/app.scss (hunk 2 above)
Comment (redox) [inline docs/src/sass/app.scss]: Sure 👍

**Pull request merged**